### DISCUSSION TITLE
repositories.yaml is not re-processed building in test or prod if repos.yaml haven't changed

### DISCUSSION THREAD

**Discussion by ssaid** [Tecnativa/doodba#522] [Doodba - The image] [answered]
Hi!
The issue I'm facing is that when running "docker-compose build" for a project that has not changed the repos.yaml the docker cache does it's job and reutilizes the cache because no files had changed so the output image does not have the code refreshed (although the repository defined in repos.yaml have newer commits of the branch configured) is this for a particular purpose ?
I'm aware that running docker-compose build --no-cache will be the solution, but what if i want to reutilize all the cache before the repos.yaml step runs?
Thank you so much for this project :clap: 
![image](https://user-images.githubusercontent.com/9728965/190495879-7dcc619c-7acf-4c5f-8fb4-cbb9ece1a542.png)

**Comment by pedrobaeza** [comment, marked as answer]
I'm afraid not. You have to do the full rebuild, but the heaviest part is the repo gathering one, so it's not a big deal.
